packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1224,6 +1224,47 @@ class EditableText extends StatefulWidget {
   ///   works only with [TextInputType.emailAddress]. Make sure the input field has a
   ///   compatible [keyboardType]. Empirically, [TextInputType.name] works well
   ///   with many autofill hints that are predefined on iOS.
+  ///
+  /// ### Troubleshooting Autofill
+  ///
+  /// Platform autofill services rely on [autofillHints] to work. Make sure the
+  /// entries in [autofillHints] are supported by the target platform, and the
+  /// autofill service currently in use (the name of the service can typically
+  /// be found in your mobile device's system settings).
+  ///
+  /// #### Autofill UI refuses to show up when I tap on the text field?
+  ///
+  /// Check if autofill is turned on on the target platform. Also make sure
+  /// there're available credentials stored in the autofill service.
+  ///
+  /// - iOS password autofill: Go to Settings -> Password, turn on "Autofill
+  /// Passwords", and add new passwords for testing by pressing the top right
+  /// "+" button. Use an arbitary "website" if you don't have associated domains
+  /// set up for your app. As long as there's at least one password stored, you
+  /// should be able to see a key-shaped icon in the quick type bar on the
+  /// software keyboard, when a password related field is focused.
+  ///
+  /// - iOS contact information autofill: iOS seems to pull contact info from
+  /// the Apple ID currently associated with the device. Go to Settings ->
+  /// Apple ID (usually the first entry, or "Sign in to your iPhone" if you
+  /// haven't set up one on the device), and fill out the relevant fields. If
+  /// you wish to test more contact info types, try adding them in Contacts ->
+  /// My Card.
+  ///
+  /// - Android autofill: Go to Settings -> System -> Languages & input ->
+  /// Autofill service. Enable the autofill service of your choice, and make
+  /// sure there're available credentials associated with your app.
+  ///
+  /// #### `TextInput.finishAutofillContext` is called but the Autofill Save
```

Comment:
nit: the docs seem to be inconsistent in the spelling of "autofill save" vs. "Autofill Save" - can we unify?

Reply:
changed to lower case, but kept some "Autofill" because those are verbatim.